pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_creating-a-product.adoc

```diff
@@ -12,23 +12,26 @@ A product must exist in {ProductName} before you can assign a product name to th
 .Procedure
 
 . Navigate to menu:Products[New Product].
-
-. Enter the following information:
-  Product Name:: The text that appears in {ProductShortName}, the document preview, and in user-facing content.
-  Product UrlFragment:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
-  Product Description (optional):: A description of the product.
-
+. Enter the following information, items marked with a star (*) are mandatory:
+[horizontal]
+  Product Name *:: The text that appears in {ProductShortName}, the document preview, and in user-facing content. This name is replaced by the value of the `:pantheonproduct:` attribute in user-facing content that {ProductShortName} generates.
+  Product URL Fragment *:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
+  Product Description:: A description of the product.
+  Product Version *:: The product version. This value is replaced by the value of the `:pantheonversion:` attribute in user-facing content that {ProductShortName} generates.
```

Comment:
> This value is replaced by the value of the `:pantheonversion:` attribute in user-facing content that {ProductShortName} generates.

This is wrong. Product version refers to the version of the Product, not Pantheon's version number. E.g. Product name = "RHEL" ; Product Version = "8.4". Or like the example farther down: `Product Version:: 4.4-Beta`

Reply:
ok, since I don't see any attributes list in our Pantheon repo, I'm simply using generic terms product version or version number in this description